examples/lib/stories/effects/opacity_effect.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:flame/components.dart';
+import 'package:flame/effects.dart';
+import 'package:flame/game.dart';
+import 'package:flame/input.dart';
+
+class OpacityEffectGame extends BaseGame with TapDetector {
+  late final SpriteComponent sprite;
+
+  @override
+  Future<void> onLoad() async {
+    final flameSprite = await loadSprite('flame.png');
+    add(
+      sprite = SpriteComponent(
+        sprite: flameSprite,
+        position: Vector2.all(100),
+        size: Vector2(149, 211),
+      ),
+    );
+
+    add(
+      SpriteComponent(
+        sprite: flameSprite,
+        position: Vector2(300, 100),
+        size: Vector2(149, 211),
+      )..addEffect(
+          OpacityEffect(
+            opacity: 0,
+            duration: 0.5,
+            initialIsInfinite: true,
```

Comment:
this is not related to this PR at all, and just an opinion:
I think this parameter should be called just `isInfinite`. I see on the impl why it's "initial" and that it can change, but the parameter name doesn't have to have the same name of the field. from the perspective of the constructor callee it feels a bit weird to me to call it "initial". thoughts?

Reply:
Yeah, I confess that this puzzled me as well, I am gonna put a trello card about this so we can discuss and do this change on a followup maybe, since this isn't directly related to this PR.

Reply:
Unsure what you mean @spydon , `initialIsInfinite` and the other property are properties from the base effect class right?

Reply:
Yeah, but they are not named arguments there. All the effects that use that call the named arguments without `initial`.

Reply:
ahhhh, got it, that slipped my attention, will change here then